src/browser/page-switch.js

```diff
@@ -31,17 +29,26 @@ define((require, exports, moudle) => {
         }
       }
     }, [
-      DOM.span({
+      DOM.div({
         key: 'thumbnail',
         className: 'tab-thumbnail',
-        style: {backgroundImage: readThumbnailURI(webViewerCursor.get('location'))},
-      }),
+      }, [
+        DOM.img({
+          key: 'image',
+          className: 'image',
+          src: thumbnail,
+          alt: '',
+          onLoad: event => URL.revokeObjectURL(event.target.src),
```

Comment:
As far as I understand that's the way blob urls are supposed to be used. Sure blob will be gc-ed but we don't care image will be rendered and isn't going to go away unless it's remove from DOM at which point we don't care.